YamlSplitter/YamlSplitter.csproj

```diff
@@ -0,0 +1,100 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
please use the same csproj format as similar to others, e.g. https://github.com/dotnet/docfx/blob/dev/tools/GEmojiCodeGenerators/GEmojiCodeGenerators.csproj

Reply:
done